src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1130,35 +1131,50 @@ public LocalDate fromString(String string) {
-    tableColumnCvrIdCol.setCellValueFactory(new PropertyValueFactory<>("idColumnIndex"));
-    tableColumnCvrPrecinctCol.setCellValueFactory(
-        new PropertyValueFactory<>("precinctColumnIndex"));
-    tableColumnCvrOvervoteDelimiter.setCellValueFactory(
-        new PropertyValueFactory<>("overvoteDelimiter"));
-    tableColumnCvrProvider.setCellValueFactory(
-        c -> new SimpleStringProperty(
-            Provider.getByInternalLabel(c.getValue().getProvider()).toString())
-    );
-    tableColumnCvrContestId.setCellValueFactory(new PropertyValueFactory<>("contestId"));
-    tableColumnCvrOvervoteLabel.setCellValueFactory(new PropertyValueFactory<>("overvoteLabel"));
-    tableColumnCvrUndervoteLabel.setCellValueFactory(new PropertyValueFactory<>("undervoteLabel"));
-    tableColumnCvrUndeclaredWriteInLabel
-        .setCellValueFactory(new PropertyValueFactory<>("undeclaredWriteInLabel"));
-    tableColumnCvrTreatBlankAsUndeclaredWriteIn
-        .setCellValueFactory(new PropertyValueFactory<>("treatBlankAsUndeclaredWriteIn"));
+    EditableColumn[] cvrStringColumnsAndProperties = new EditableColumn[]{
+        new EditableColumnString(tableColumnCvrFilePath, "filePath"),
+        new EditableColumnString(tableColumnCvrFirstVoteCol, "firstVoteColumnIndex"),
+        new EditableColumnString(tableColumnCvrFirstVoteRow, "firstVoteRowIndex"),
+        new EditableColumnString(tableColumnCvrIdCol, "idColumnIndex"),
+        new EditableColumnString(tableColumnCvrPrecinctCol, "precinctColumnIndex"),
+        new EditableColumnString(tableColumnCvrOvervoteDelimiter, "overvoteDelimiter"),
+        new EditableColumnString(tableColumnCvrProvider, "provider"),
+        new EditableColumnString(tableColumnCvrContestId, "contestId"),
+        new EditableColumnString(tableColumnCvrOvervoteLabel, "overvoteLabel"),
+        new EditableColumnString(tableColumnCvrUndervoteLabel, "undervoteLabel"),
+        new EditableColumnString(tableColumnCvrUndeclaredWriteInLabel,
+           "undeclaredWriteInLabel"),
+        new EditableColumnBoolean(tableColumnCvrTreatBlankAsUndeclaredWriteIn,
+           "treatBlankAsUndeclaredWriteIn"),
+    };
+    setUpEditableTableStrings(cvrStringColumnsAndProperties);
+
+    // Don't allow editing of Provider to avoid the complexity of creating a dropdown here,
+    // it's probably not going to be frequently edited anyway, and the UI for Provider selection
+    // is cleaner anyway since it nicely disables and enables supported fields.
+    tableColumnCvrProvider.setEditable(false);
+
     tableViewCvrFiles.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
-    tableViewCvrFiles.setEditable(false);
+    tableViewCvrFiles.setEditable(true);
+
+    EditableColumn[] candidateStringColumnsAndProperties = new EditableColumn[]{
+        new EditableColumnString(tableColumnCandidateName, "name"),
+        new EditableColumnList(tableColumnCandidateAliases, "aliases"),
```

Comment:
We talked about having this work as a pop-up in [this thread](https://github.com/BrightSpots/rcv/pull/639/files#r1141150539). Right now, there's zero guidance to the user that they need to put semicolons in there if they expect to break out separate aliases.

A seriously confusing UX is, if they enter aliases with semicolons in the box above the table ( (e.g. "test1;test2" or even "test1; test2"), it doesn't treat them as separate aliases (you can verify this because validation passes); the expectation there is that it's supposed to be entered with a line break.

However, if they then double-click the cell and press enter, it will subsequently convert them into separate aliases.

Reply:
Nice, I think that's a big improvement! One thing I noticed that's still a bit off is if you end a line with a semi-colon... it kind of just vanishes after you commit it.

Reply:
Good catch -- that's an issue with the existing aliases code. Split that out just to make sure: https://github.com/BrightSpots/rcv/pull/726

Reply:
Scratch my previous [deleted] comment.

It's an issue in the existing code too, when creating aliases, and when editing. Fixed for both -- I had the regex eat all non-word characters `\W` instead of all non-whitespace characters `\s`

Reply:
Seems to work well now, nice work.